src/todo/todo.lua

```diff
@@ -187,17 +189,16 @@ end
 
 function todo.mark_complete(id)
     todo.log("Marking task [" .. id .. "] as completed.")
-    local t
     for i, task in ipairs(global.todo.open) do
         if (task.id == id) then
-            t = table.remove(global.todo.open, i)
+            local t = table.remove(global.todo.open, i)
             todo.log("Removed task from open list.")
+
+            todo.log("Adding task [" .. t.id .. "] to done list.")
+            table.insert(global.todo.done, t)
```

Comment:
Good thing done here.

Reply:
Yeah.. I found that, did a git blame, found my name and was like "what the fuck was I thinking?" :D